assets/queries/k8s/saTokenSecretQuery/query.rego

```diff
@@ -0,0 +1,56 @@
+package Cx
+
+CxPolicy [result] {
+    document := input.document[i]
+    metadata := document.metadata
+    
+    validKind := ["Role","ClusterRole"]
+    ruleTaint := ["get","watch","list","*"]
+    resourcesTaint := ["secrets","pods"]
+    
+    kind := document.kind
+    contains(validKind,kind)
+    name := metadata.name
+    
+    bindingExists(name,kind)
+    
+   	resources := document.rules[_].resources; 
+    some resource
+    	contains(resourcesTaint,resources[resource])
+    
+    rules := document.rules[_].verbs;    
+    some rule
+    	contains(ruleTaint,rules[rule])
+    
+    result := {
+                "documentId":        input.document[i].id,
+                "searchKey":         sprintf("metadata.name=%s.rules.verbs", [metadata.name]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue":  sprintf("The resources %s should not contain the following verbs: [%s]", [resources,rules]),
+                "keyActualValue":    sprintf("The resources %s should contain the following verbs: [%s]", [resources,rules])
```

Comment:
The keyExpected/Actual should specify the path of the attribute where the wrong/missing value is